.ci/on-schedule.sh

```diff
@@ -107,6 +109,46 @@ function collect_aur_timestamps() {
     http_proxy="$CI_AUR_PROXY" https_proxy="$CI_AUR_PROXY" UTIL_FETCH_AUR_TIMESTAMPS collect_aur_timestamps_output "${AUR_PACKAGES[*]}"
 }
 
+function collect_changed_libs() {
+    for repo in "${CI_ARCH_REPOS[@]}"; do
+        UTIL_PARSE_DATABASE "$repo" "${CI_ARCH_MIRROR}/$repo/os/x86_64/${repo}.db"
+    done
+    UTIL_PARSE_DATABASE "${REPO_NAME}" "${CI_CAUR_MIRROR}/${REPO_NAME}/x86_64/${REPO_NAME}.db"
+
+    diff .ci/lib.state .ci/lib.state.new | grep -E '^< ' | cut -d' ' -f2- >.ci/lib.state.diff
+    while IFS= read -r line; do
+        IFS=':' read -r -a pkg <<<"$line"
+        CHANGED_LIBS+=("${pkg[0]}")
+    done <.ci/lib.state.diff
+}
+
+function update-lib-bump() {
+    local bump=0
+    local -n CONFIG=${1:-VARIABLES}
```

Comment:
This variable should probably not be CONFIG, but something more distinct to avoid potential issues.